### PR TITLE
Separating the logic up into multiple files

### PR DESCRIPTION
**Pull request by padster**

### CODE REVIEW

**Comment by padster** [comment]
Added the missing files

File: bot.js [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
-});
-bot.on('ready', function (evt) {
-    logger.info('Connected');
-    logger.info('Logged in as: ');
-    logger.info(bot.username + ' - (' + bot.id + ')');
-});
 
-bot.on('message', function (discordName, discordID, channelID, message, evt) {
+// Add message handling for the bot:
+dicordClient.on('message', function (discordName, discordID, channelID, message, evt) {
```
Comment (prayansh) [inline bot.js]: wouldn't this be `bot.discordClient`?

File: bot.js [hunk 2]
```diff
@@ -1,38 +1,23 @@
...
     // Our bot needs to know if it will execute a command
     // It will listen for messages that will start with `!`
     if (message.substring(0, 1) == '!') {
```
Comment (prayansh) [inline bot.js]: can u also change this to `===` here?

File: bot.js (hunk 1 above)
Comment (prayansh) [reply bot.js]: Also i think this is a typo

**Comment by padster** [comment]
Typos fixed, looks like it's working again.

**Review by prayansh — APPROVED** [review]

**Pull request merged**